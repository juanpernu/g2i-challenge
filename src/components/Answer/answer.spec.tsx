import { create } from 'react-test-renderer';
import { Answer } from './index';

describe('<Question />', () => {
  const Mock = {
    question: {
      category: 'Test question',
      question: 'Is this a test question?',
      correct_answer: 'True',
      difficulty: 'hard',
      incorrect_answers: ['False'],
      type: 'boolean'
    },
    answer: 'True'
  };

  it('should match snapshot', () => {
    const component = create(
      <Answer trivia={Mock.question} answer={Mock.answer} />
    );
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
