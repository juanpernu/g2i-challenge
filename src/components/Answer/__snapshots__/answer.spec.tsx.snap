// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Question /> should match snapshot 1`] = `
<div
  className="mb-2 flex flex-col items-center p-4 bg-green-100"
>
  <svg
    aria-hidden="true"
    className="h-5 w-5 text-green-500"
    fill="currentColor"
    viewBox="0 0 20 20"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clipRule="evenodd"
      d="M10 18a8 8 0 100-16 8 8 0 000 16zm3.707-9.293a1 1 0 00-1.414-1.414L9 10.586 7.707 9.293a1 1 0 00-1.414 1.414l2 2a1 1 0 001.414 0l4-4z"
      fillRule="evenodd"
    />
  </svg>
  <p
    className="text-slate-600 text-base my-2"
  >
    Is this a test question?
  </p>
</div>
`;
